tool/caffe_model/caffe_parse/caffe.proto

```diff
@@ -0,0 +1,1399 @@
+syntax = "proto2";
```

Comment:
move this into src/proto/caffe.proto?
better add the commit of caffe from which the caffe.proto is copied.